### ISSUE TITLE
Workspace

### ISSUE THREAD

**Issue by williantenfen** [pgmodeler/pgmodeler#849]
Is there a way to create 'Workspaces' for a data model? So we can select only entities and relationships that we need to work besides having all entities showing on same screen?
Thanks.

**Comment by rkhaotix** [comment]
No. There is no such feature. Currently, all objects are exposed to the
user.

Em 23 de jul de 2016 17:25, "Willian Tenfen Wazilewski" <
notifications@github.com> escreveu:

**Comment by williantenfen** [comment]
Any chance to have this on future? How do I set this issue as a suggestion? Should I close it? Thanks...
